backend/Epicenter.Domain/Models/DTO/MissingModelResponse.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Epicenter.Domain.Models.DTO
+{
+    public class MissingModelResponse
+    {
+        public string FirstName { get; set; }
```

Comment:
not enough to return these properties, we need whole missingmodel object with id, etc.